lib/DriverTool/swift_parse_test_main.cpp

```diff
@@ -54,6 +59,9 @@ struct SwiftParseTestOptions {
       "skip-bodies",
       llvm::cl::desc("skip function bodies and type members if possible"));
 
+  llvm::cl::opt<bool> Dump = llvm::cl::opt<bool>(
+      "dump", llvm::cl::desc("dump result for each iteration"));
```

Comment:
What exactly is this supposed to dump? The AST?

Reply:
Whatever the result. for libParse and ASTGen, the result is AST. For SwiftParser, I'm planning to just dump the syntax tree. It's just convenient to confirm that it actually performed the parsing.

Reply:
Ah, OK 👍🏽